pkg/sql/randgen/schema.go

```diff
@@ -17,7 +17,7 @@ import (
 	"math/rand"
 	"strings"
 
-	"github.com/cockroachdb/apd/v3"
+	apd "github.com/cockroachdb/apd/v3"
```

Comment:
is this `apd` change necessary?

Reply:
my editor does it automatically when the path doesn't match the package name